crates/ruff/src/autofix/edits.rs

```diff
@@ -179,36 +175,29 @@ pub(crate) fn remove_argument(
     }
 }
 
-/// Determine if a body contains only a single statement, taking into account
-/// deleted.
-fn has_single_child(body: &[Stmt], deleted: &[&Stmt]) -> bool {
-    body.iter().filter(|child| !deleted.contains(child)).count() == 1
+/// Determine if a vector contains only one, specific element.
+fn is_only<T: PartialEq>(vec: &[T], value: &T) -> bool {
+    vec.len() == 1 && &vec[0] == value
```

Comment:
Nit: Does `matches!(vec.as_slice(), [value])` work?

Reply:
No, `matches!(vec, [v] if v == value)` does, though is that better?